grails-app/services/de/iteratec/osm/result/ApplicationDashboardService.groovy

```diff
@@ -118,4 +126,29 @@ class ApplicationDashboardService {
 
     }
 
+    List<Map> getAllActivePagesAndMetrics(Long jobGroupId) {
+        List<Map> recentMetrics = getRecentMetricsForJobGroup(jobGroupId).collect {
+            it.projectedProperties.pageName = Page.findById(it.pageId).name
+            return it.projectedProperties
+        }
+
+        getPagesOfActiveJobs(jobGroupId).each {
+            Page page = it
+            if (page.name != Page.UNDEFINED &&
```

Comment:
filter the list with this condition before the each expression, please.

Reply:
done